.rhcicd/floorist/floorplan.yaml

```diff
@@ -38,107 +38,55 @@ objects:
             event_type_behavior AS etb
           INNER JOIN
             event_type AS et
-             ON et.id = etb.event_type_id
+              ON et.id = etb.event_type_id
           INNER JOIN
             applications AS apps
-              ON apps.id = et.application_id 
+              ON apps.id = et.application_id
           INNER JOIN
             behavior_group AS bg
               ON bg.id = etb.behavior_group_id
           INNER JOIN
             bundles AS bun
               ON bun.id = bg.bundle_id
-          WHERE
-            bg.org_id NOT IN (${FLOORIST_INTERNAL_ORG_IDS_FILTER})
           GROUP BY
             bun.display_name,
             apps.display_name,
             et.display_name,
+            bg.org_id,
             actively_used
-      # Count the number of behavior groups associated to event types per
-      # organization, and also show the bundle and the application they are
-      # associated to.
-      - prefix: insights/notifications/behavior_groups_event_types_org_id
+        # Count the number of email subscriptions, and group them by
+        # application, subscription type, whether they are subscribed or not
+        # and their org id.
+      - prefix: insights/notifications/email_subscriptions_by_org_id
```

Comment:
Email subscription by default from event type doesn't seems covered here, is it on purpose ?

Reply:
I just moved one of the queries but you might have spotted something that I missed. I don't follow you, though. What do you mean by the email subscription by default? Are you referring to `endpoint_email_subscriptions`, maybe? :confused: 

Thanks!

Reply:
I'm refering to column `subscribed_by_default` from db table `event_type` that we use for OCM.
If this request should reflect email subscriptions **choosen** by users, then it's correct.

Reply:
Hmmmmm... so we would have to check the notification history for those events that have an event type which is subscribed by default, no? However wouldn't that give us some duplicates?

Like if I am a new user, I will be subscribed by default to event A which has that option switched on. Therefore, there won't be a record in the `email_subscriptions` table, right?

What if that user disables that subscription manually? I guess we will create an unsubscription record. But if the user turns it back on... Do we actually remove the unsubscription?

Reply:
In a second thought, events subscribed by default can't be only our database data.
Because unless users unsubscribed from this event, it will be sent to all members of an org, even if they never went to Notifications.
Building those statistics will require to fetch all users of all orgs then removing users how unsubscribed explicitly.
I think (I hope) it's out of scope :)